src/test/objectstore/store_test.cc

```diff
@@ -5901,7 +5909,9 @@ TEST_P(StoreTest, garbageCollection) {
       ASSERT_EQ(r, 0);
       ASSERT_EQ(statfs.compressed_allocated, 0x20000);
       const PerfCounters* counters = store->get_perf_counters();
+#if defined(HAVE_LIBAIO)
```

Comment:
this test is only valid with bluestore, could just quote it using `#ifdef HAVE_LIBAIO`